eZ/Publish/Core/settings/repository.yml

```diff
@@ -1,69 +1,46 @@
 services:
-    # API Aliases
-    ezpublish.api.repository:
-        alias: ezpublish.signalslot.repository
-        public: true
-
-    ezpublish.api.service.bookmark:
-        alias: ezpublish.signalslot.service.bookmark
-        public: true
-
-    ezpublish.api.service.content:
-        alias: ezpublish.signalslot.service.content
-        public: true
-
-    ezpublish.api.service.content_type:
-        alias: ezpublish.signalslot.service.content_type
-        public: true
-
-    ezpublish.api.service.field_type:
-        alias: ezpublish.signalslot.service.field_type
-        public: true
-
-    ezpublish.api.service.role:
-        alias: ezpublish.signalslot.service.role
-        public: true
-
-    ezpublish.api.service.object_state:
-        alias: ezpublish.signalslot.service.object_state
-        public: true
-
-    ezpublish.api.service.url_wildcard:
-        alias: ezpublish.signalslot.service.url_wildcard
-        public: true
-
-    ezpublish.api.service.url_alias:
-        alias: ezpublish.signalslot.service.url_alias
+    _defaults:
         public: true
 
-    ezpublish.api.service.user:
-        alias: ezpublish.signalslot.service.user
-        public: true
-
-    ezpublish.api.service.search:
-        alias: ezpublish.signalslot.service.search
-        public: true
-
-    ezpublish.api.service.section:
-        alias: ezpublish.signalslot.service.section
-        public: true
-
-    ezpublish.api.service.trash:
-        alias: ezpublish.signalslot.service.trash
-        public: true
-
-    ezpublish.api.service.location:
-        alias: ezpublish.signalslot.service.location
-        public: true
-
-    ezpublish.api.service.language:
-        alias: ezpublish.signalslot.service.language
-        public: true
-
-    ezpublish.api.service.url:
-        alias: ezpublish.signalslot.service.url
-        public: true
-
-    ezpublish.api.service.notification:
-        alias: eZ\Publish\Core\SignalSlot\NotificationService
-        public: true
+    # API Aliases
+    eZ\Publish\API\Repository\Repository:
+        alias: eZ\Publish\Core\SignalSlot\Repository
```

Comment:
What's the difference between this form and the shorter form below?

Reply:
> What's the difference between this form and the shorter form below?

There's no difference, good catch, thanks :) 
I'll make it shorter when back on this issue.